components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```diff
@@ -839,26 +844,28 @@ def main(argv):
         conf_dir = clp_home / "etc"
 
         # Start components
-        if "" == component_name or DB_COMPONENT_NAME == component_name:
+        LAUNCH_ALL = ""
```

Comment:
* Did you mean to replace all the `""` with this variable?
* Perhaps rename to `ALL_TARGET`?